### DISCUSSION TITLE
Throwing 403 FORBIDDEN error on production

### DISCUSSION THREAD

**Discussion by s1rbl4ck** [MohmmedAshraf/laravel-translations#27] [Q&A] [answered]
Hello hope you're doing well, everything working well on local but when I deploy it it's throwing 403 error on production...
I should also mention that I deploy the project on a docker container...

**Comment by MohmmedAshraf** [comment, marked as answer]
Hi @s1rbl4ck it should be because you have to set your rules for who can access it, Please check the [Authorization](https://github.com/MohmmedAshraf/laravel-translations#authorization) section.
